libs/model/src/models/comment.ts

```diff
@@ -76,63 +74,46 @@ export default (
         afterCreate: async (comment: CommentInstance, options) => {
           const { Thread, Outbox } = sequelize.models;
           const thread_id = comment.thread_id;
-          try {
-            const thread = await Thread.findOne({
-              where: { id: thread_id },
+          const thread = await Thread.findOne({ where: { id: thread_id } });
```

Comment:
Can't we also optimize this with:
```
await Thread.update(
  {
    comment_count: Sequelize.literal('comment_count + 1'),
    activity_rank_date: comment.created_at,
  },
  {
    where: {
      thread_id,
    },
    transaction: options.transaction,
  },
);
```
Can then move:
```
stats().increment('cw.hook.comment-count', {
  thread_id: String(thread_id),
});
```
to the end since the transaction will throw if the thread does not exist anyway (due to `Comment.thread_id` foreign key)

Reply:
just pushed a commit with multiple hook optimizations... this was not in scope but makes sense to optimize

Reply:
@timolegros are you reviewing or should I merge? Don't want to get your feedback after merging again 😆